configuration/pih/liquibase/liquibase.xml

```diff
@@ -832,16 +832,6 @@
             delete from global_property where property = 'openconceptlab.oclLoadAtStartupPath';
         </sql>
     </changeSet>
-    
-    <changeSet id="20240122-migrate-noncoded-dxs" author="ddesimone">
-        <preConditions onFail="MARK_RAN">
-            <sqlCheck expectedResult="1">
-                select count(*) from address_hierarchy_entry where parent_id is NULL and name like 'Haiti';
-            </sqlCheck>
-        </preConditions>
-        <comment>UHM-7593: Migrate ZL MCH non-coded to coded diagnoses (January 2024)</comment>
-        <sqlFile endDelimiter=";" path="sql/migrate_non_coded_mch_dxs.sql" relativeToChangelogFile="true" stripComments="true"/>
-    </changeSet>
```

Comment:
Why did you need to delete this and create a new changeset?  Did you check to see what happens if you just modify this one?  Worst case you need to add a valid md5sum, right?